### DISCUSSION TITLE
Spring Boot에서 Redis를 활용한 검색 속도 최적화 연구: 자주 검색되는 주식 데이터를 캐싱하여 성능 개선 방안 탐구

### DISCUSSION THREAD

**Discussion by bluegown** [100-hours-a-week/discussions#45] [Deepdive]
주식을 검색하는 과정에서 , 자주 검색되는 것들이 빠르게 노출될 수 있도록 합니다.

- Redis를 사용해 자주 검색되는 데이터를 캐싱하여 검색 속도를 높이는 방법
- 캐시 미스(cache miss) 시 MariaDB에서 데이터를 조회하여 캐시에 저장

검색 기능을 구현하면서 , Redis의 캐시를 효율적으로 사용하는 방법에 대해서 배우려고 합니다

**Comment by startupcodekr** [comment]
Spring Boot에서 Redis를 활용한 검색 속도 최적화 연구: 자주 검색되는 주식 데이터를 캐싱하여 성능 개선 방안 탐구
